wiki/People/Beatmap_Nominators/Beatmap_Veto/en.md

```diff
@@ -0,0 +1,56 @@
+
+## Performing a beatmap veto
+
+As a Beatmap Nominator, you can veto a beatmap in one of two ways: 
+
+* On older beatmaps with forum threads, place a Bubble Pop icon
+* On newer beatmaps with discussions, post a Problem stamp
+
+In both cases, the beatmap veto should be stated as such, and should be accompanied by a detailed explanation as to why you believe the beatmap in question should not be considered for Qualification.
+
+A proper discussion which attempts to reach a mutual agreement by both sides of the argument must then follow.
+
+Assuming the issues are satisfactorily resolved or justified with all parties in agreement, the Nominator may lift the beatmap veto and allow Qualification to proceed.
+
+## What happens if no agreement is reached?
+
+Mapping is extremely diverse and creative differences are to be expected. This is to be encouraged for the sake of creating a variety of content all players can enjoy! However, such differences can naturally lead to disagreements.
+
+Where neither party can agree to compromise on an issue, the Quality Assurance Team will act as mediators to decide if the veto should be dismissed or upheld.
+
+If discussion is no longer progressing after 48 hours have passed, the mapper or Nominator(s) can request for mediation. If no request is received, this mediation will begin after one week, regardless of whether discussion is still ongoing or not.
+
+During mediation, the disqualification branch of the QAT will discuss the beatmap and reasons for and against the veto. This includes an evaluation as to whether the proposed changes are deemed necessary and/or beneficial. Mediation will last for no longer than three days.
+
+A decision to uphold or dismiss will be based on majority vote, with a minimum of either seven branch members, or half of the total members participating.
+
+## How to proceed after mediation
+
+Following mediation, the beatmap veto will either be dismissed if the proposed changes are deemed unnecessary or detrimental to the beatmap, or upheld if they are considered necessary or beneficial.
+
+### Dismissed beatmap vetoes
+
+**If the beatmap veto is dismissed**, any formally qualified Beatmap Nominator may renominate the beatmap - including the Nominator who initially nominated.
+
+A previously uninvolved Beatmap Nominator can propose to re-initiate mediation at any point after giving their thoughts on the thread/discussion. This mediation will be evaluated as previously, taking into account the time since making the previous decision and reason given.
+
+Beatmap Nominators may no longer overrule a beatmap veto, and must instead re-initiate mediation with the beatmap veto passing a dismissal before both they and the initial Beatmap Nominator can nominate (should they wish to do so).
```

Comment:
> Beatmap Nominators may no longer overrule a beatmap veto

So, is this saying that they were able to, or they were never allowed to do so in the first place?

Reply:
Vetoes could previously be overruled by a third, previously uninvolved Nominator if they didn't agree with the reasons for the veto